indexing-service/src/test/java/org/apache/druid/indexing/common/task/concurrent/ActionsTestTask.java

```diff
@@ -101,28 +121,10 @@ public SegmentIdWithShardSpec allocateSegmentForTimestamp(DateTime timestamp, Gr
             TaskLockType.APPEND
         )
     );
-  }
-
-  public SegmentIdWithShardSpec allocateSegmentForTimestamp(
```

Comment:
Was this method not being used anywhere?

Reply:
No